storage/innobase/xtrabackup/doc/source/xtrabackup_bin/scripting_backups_xbk.rst

```diff
@@ -1,23 +1,39 @@
 .. _scripting-xtrabackup:
 
+================================================================================
 Scripting Backups With xtrabackup
```

Comment:
this section should be removed as it only made sense when xtrabackup and innobackupex were separate and innobackupex used suspend to copy .frm files.